src/pybind/mgr/dashboard/controllers/cephfs.py

```diff
@@ -6,33 +6,36 @@
 import cherrypy
 
 from ..exceptions import DashboardException
-from . import ApiController, AuthRequired, Endpoint, BaseController
+from . import ApiController, AuthRequired, Endpoint, RESTController
 from .. import mgr
 from ..services.ceph_service import CephService
 from ..tools import ViewCache
 
 
 @ApiController('/cephfs')
 @AuthRequired()
-class CephFS(BaseController):
+class CephFS(RESTController):
     def __init__(self):
         super(CephFS, self).__init__()
 
         # Stateful instances of CephFSClients, hold cached results.  Key to
         # dict is FSCID
         self.cephfs_clients = {}
 
-    @Endpoint()
-    def clients(self, fs_id):
+    def list(self):
+        fsmap = mgr.get("fs_map")
+        return fsmap['filesystems']
+
+    def get(self, fs_id):
         fs_id = self.fs_id_to_int(fs_id)
 
-        return self._clients(fs_id)
+        return self.fs_status(fs_id)
 
     @Endpoint()
-    def data(self, fs_id):
+    def clients(self, fs_id):
```

Comment:
I think what we should use here is a `@RESTController.Resource('GET')` decorator, because we want to get the list of clients for the particular filesystem resource with id `fs_id`. The URL for this endpoint then would be `/cephfs/{fs_id}/clients`